README.md

```diff
@@ -122,6 +122,20 @@ Pro-tip: we have had at least one large code refactor. You can ignore specific l
 git config blame.ignoreRevsFile .git-blame-ignore-revs
 ```
 
+### Google Credentials
+
+You might need to provide GCP credentials for the app to use.
```

Comment:
We could maybe get slightly more descriptive here, "After an experiment is complete, on certain pages an API endpoint is called to receive experiment analysis data from Jetstream to display visualization tables. To see experiment visualization data, you must provide GCP credentials."

Reply:
I'm happy to put this exact phrase since I don't know a whole lot about where else GCP credentials might be needed. Is it just for accessing visualization data?

Reply:
@emtwo can probably confirm/check my wording but I don't believe we need it for anything else.

Reply:
For now, it's not needed for anything else. I do think this phrasing might be a bit *too* specific though: `After an experiment is complete...`. Only for now it's after completion. There will later be data available live too. And who knows where else we might decide to put experiment results data later on! DATA EVERYWHERE!!!! 
![image](https://user-images.githubusercontent.com/784781/98976397-4d6c8380-24e5-11eb-9e5c-3ad59d82c0a4.png)

Reply:
That could maybe be tweaked then @jodyheavener, sorry 🤔 Going to r+ though, merge at will!